package.json

```diff
@@ -34,14 +34,15 @@
     "lib/contexts"
   ],
   "dependencies": {
-    "credentials-context": "^1.0.0",
     "commander": "^2.20.3",
+    "credentials-context": "^1.0.0",
     "debug": "^4.1.1",
     "fs-extra": "^8.1.0",
     "get-stdin": "^7.0.0",
     "jsonld": "^2.0.2",
     "jsonld-signatures": "^5.0.0",
-    "supports-color": "^7.1.0"
+    "supports-color": "^7.1.0",
+    "veres-one-context": "^11.0.0"
```

Comment:
Is this context used outside the test suite?  If not, please move to dev-dependencies.